settings.gradle.kts

```diff
@@ -16,13 +16,21 @@ dependencyResolutionManagement {
             url = URI("https://jitpack.io")
         }
         maven {
-            name = "GitHubPackages"
+            name = "arklib-android GitHub Packages"
             url = URI("https://maven.pkg.github.com/ARK-Builders/arklib-android")
```

Comment:
Let's use `ark-android` to avoid confusion with the archived repo.

It was just a fat lib when we were prototyping, now we moved to the next level with modular framework 😁

Reply:
Yes. I don't know if I understood it correctly. This is indeed a reference to `arklib-android` archive, it is not a name for the current repo. Should we use another repo instead?

Reply:
Oh, I didn't notice the link leads to the archived repo.

We certainly should not use archived repos.

Reply:
But what do we use from `arklib-android`?

Reply:
That's a good question, I've just realized we're actually not using it at all

Reply:
@kirillt ACTUALLY, I take back what I said :laughing:

It's being used in at least 23 places, just search for `dev.arkbuilders.arklib` and you'll realize it too. Here's an example on the `sample` package https://github.com/ARK-Builders/ark-android/blob/9ab96f3d9494cc7489241fe5b21b3cc8f2597181/sample/src/main/java/dev/arkbuilders/sample/App.kt#L4-L6

Reply:
> But what do we use from `arklib-android`?

`TagSelector` and `ScoreWidget` dependent on arklib, they need `ResourceIndex`, `TagStorage`, `ScoreStorage` and so on

Reply:
Thanks guys, I wasn't realizing it.. I'll come up with plan for the upgrade later.